src/vsphere_cpi/lib/cloud/vsphere/cloud.rb

```diff
@@ -18,6 +18,23 @@ def message
     attr_accessor :client, :logger # exposed for testing
     attr_reader :config, :datacenter, :heartbeat_thread
 
+    def add_cpi_telemetry_config
+      # Add telemetry option to vsphere
+      cpi_advanced_prop=VimSdk::Vim::Option::OptionValue.new
+      cpi_advanced_prop.key='config.SDDC.cpi'
+      cpi_advanced_prop.value='true'
+      begin
+        client.service_content.setting.query_view(cpi_advanced_prop.key)
+      rescue StandardError => e
```

Comment:
shoudl we be catching Exception over here instead (std err ancestors `[StandardError, Exception, Object, Kernel, BasicObject]`)? it may cover more cases that we should ignore.